README.md

```diff
@@ -78,16 +88,17 @@ need to use Django admin for the next step anyway.
 #### 4) Add an OAuth client
 
 Run Django admin (see "In Django admin" section from step 2), navigate to the
-OAuth2 clients section (eg: http://192.168.33.10:8000/admin/oauth2/client/), and add a
-new client. Fill in the values as follows:
+Django OAuth Toolkit section (/admin/oauth2_provider/), and add a
+new Application. Fill in the values as follows:
 
 - **User**: Use the lookup (magnifying glass) to find your superuser
-- **Name**: Anything you want. Something like 'micromasters-local'
-- **Url**: The URL where MicroMaster’s will be running. If you're running it via
-Docker, run ``docker-machine ip`` from the host machine to get the container IP.
-MicroMaster’s runs on port ``8079`` by default, so this value should be something
-like ``http://192.168.99.100:8079``
-- **Redirect uri**: Your **Url** value with "/complete/edxorg/" at the end
+- **Redirect uris**: The URL where MicroMaster’s will be running, followed by "/complete/edxorg/".
+If you're running it via Docker, run ``docker-machine ip`` from the host machine to get the
+container IP. MicroMaster’s runs on port ``8079`` by default, so this value should be something
+like ``http://192.168.99.100:8079/complete/edxorg/``
```

Comment:
I believe this IP is only correct if you're on a Mac and using docker-machine?

@aliceriot @noisecapella do you use localhost for the redirect uri?